app/assets/javascripts/components/filters.js.es6

```diff
@@ -1,74 +1,70 @@
-var store = {
-  state: {
-    categories: []
-  }
-};
-
+//--------------------------------------------------------------------------------
+// EventBus listener
+//--------------------------------------------------------------------------------
+window.EventBus = new Vue();
+
+//--------------------------------------------------------------------------------
+// filters
+//--------------------------------------------------------------------------------
 Vue.component('filters', {
-
   template: `
```

Comment:
Did we get this working with backticks in the end? My memory is terrible for a tuesday